### PR TITLE
89 husky install deprecated

### LINKED ISSUE #89: Husky install deprecated
# Is this worth fixing?

image

image

## So `husky init` isn't an option.
According to the [How To Section # Manual Setup](https://typicode.github.io/husky/how-to.html#manual-setup)
image

Current `package.json`:

```json
"scripts": {
  "prepare": "husky install",
}
```
Should be changed to:

```json
"scripts": {
  "postinstall": "husky",
}
```
or simply:

```json
"scripts": {
  "prepare": "husky",
}
```
since it seems to work fine. If not, I guess the README.md would need to be updated as well:

image

We already have this with `yarn pre-commit` instead

image

Thoughts...

### PR DESCRIPTION
**Pull request by GMSoudersJr**
## Previously

image

## Currently

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ To view our site and see your contributions, visit <https://fcc-seoul.fly.dev>
 - Husky setup (only once)
 
   ```bash
-  yarn prepare
+  yarn run postinstall
```
Comment (ivanms1) [inline README.md]: I think you don't need `run` here, but not 100% sure.

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```
Comment (ivanms1) [inline .husky/pre-commit:3]: not opposed to it, but can I know the reason for removing this?

**Review by ivanms1 — APPROVED** [review]
Thank you!
just a small suggestion and a question.

File: .husky/pre-commit (hunk 2 above)
Comment (GMSoudersJr) [reply .husky/pre-commit:3]: sure.

> # And there's more
> image

This was another message that appeared when running husky. I originally posted this in the issue. I guess it is better to include it in the PR next time.

File: README.md (hunk 1 above)
Comment (GMSoudersJr) [reply README.md]: okay, neither am I. I was just following husky's docs.

image

After looking through the yarn docs:
image

Okay, I will get rid of it.

**Comment by GMSoudersJr** [comment]
# 🤔 Is `yarn postinstall` needed in `README.md` anymore?

## After reading more of the docs, I came across this: *post*install will run directly after install. No further commands needed.
image


## I ran `yarn`, which equates to `yarn install` and well, here's the output:
image

I thought it might be because I already have the project and ran husky before. So, I:
1. Created a new directory
2. Cloned this repo into it
3. Went into that directory
4. `git switch` this branch
5. `yarn` then...
image

Changed some code and the linter ran as usual. 👍

File: README.md (hunk 1 above)
Comment (drecali) [reply README.md]: This seems legit to me

**Review by drecali — APPROVED** [review]
Looks good to me 🧶 Thanks!

**Pull request merged**